lesson_7/mihaela_vaida/print_register_dec.py

```diff
@@ -0,0 +1,21 @@
+
+print_registry = []
+def register(fnc):
+    
+    
+    def print_register(*args):        
+        print_registry.append(fnc.__name__)
+        print(print_registry)
+    return print_register   
+        
+
+
+@register
```

Comment:
Two lines are recommended between functions.

Reply:
I will correct that :)